### PR TITLE
Some pt_PT localization

### PR DESCRIPTION
**Pull request by ambs**
Please confirm the deletion of localization entries on those files. Also, not sure how to continue sending PRs while keeping my branch on ojs_3_1_1.

### CODE REVIEW

**Comment by ambs** [comment]
I noticed some Brazilian Portuguese on some entries. I did not read them all, so probably I will find some further fixes. But this completes all the localization files in this folder.

**Comment by asmecher** [comment]
Thanks, @ambs! Are you still working on these, or should I merge them now?

As for keeping synchronization of `ojs-stable-3_1_1` and `master`, send your PRs to master, and while it's easy to do we'll cherry-pick them back to `ojs-stable-3_1_1`.

**Comment by ambs** [comment]
I intend to work in the plugins in the next days, so if you prefer, just wait a little more :) Thanks

**Comment by ambs** [comment]
@asmecher this should be done for PKP-lib. Please wait for Travis to see if I managed to correctly define the DTDs ;)

File: plugins/generic/usageStats/locale/pt_PT/locale.xml [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+	<message key="plugins.generic.usageStats.settings.statsDisplayOptions.chartType">Escolha o tipo de gráfico para apresentar as estatísticas de download</message>
+	<message key="plugins.generic.usageStats.settings.statsDisplayOptions.chartType.bar">Barra</message>
+	<message key="plugins.generic.usageStats.settings.statsDisplayOptions.chartType.line">Linha</message>
+	<message key="plugins.generic.usageStats.settings.statsDisplayOptions.datasetMaxCount">Definir o número máximo de dados para apresentar ao mesmo tempo para um ponto específico do eixo X. Um valor mais alto pode gerar gráficos difíceis de entender. Um valor entre 3 e 5 é adequado.</message>
+	<message key="plugins.generic.usageStats.statsSum">Somar todos os downloads de ficheiros</message>
+	<message key="plugins.generic.usageStats.noStats">Não há dados estatísticos.</message>
+	<message key="plugins.generic.usageStats.monthInitials">Jan Fev Mar Abr Mai Jun Jul Ago Set Out Nov Dez</message>
+  <message key="plugins.generic.usageStats.settings.statsDisplayOptions.contextWide">Estas propriedades só serão aplicadas às estatísticas de uso em {$contextName}.</message>
```
Comment (asmecher) [inline plugins/generic/usageStats/locale/pt_PT/locale.xml]: A couple of space-based indents crept in here.

**Comment by asmecher** [comment]
Travis testing behaves a little differently when it comes to `pkp-lib` -- you'd need to send in a pull request to `ojs` with an update to the `lib/pkp` submodule. I've validated locally and everything looks fine. There are a couple of space-based indents -- I'll fix these myself after merging. Thanks for contributing!

**Comment by ambs** [comment]
Nice, thank you!

**Pull request merged**